### PR TITLE
Added Checkers, Getting Ready for P2 Release

### PR DESCRIPTION
**Pull request by yuce**
- Fixed staticcheck warnings
- Re-enabled heartbeats

### CODE REVIEW

File: internal/serialization/object_data_test.go [hunk 1]
```diff
@@ -214,9 +203,9 @@ func TestObjectDataInput_ReadUTF2(t *testing.T) {
 	o := NewObjectDataOutput(0, nil, false)
 	o.WriteString("Furkan Şenharputlu")
 	o.WriteString("Jack")
-	o.WriteString("⚐中💦2😭‍🙆😔")
```
Comment (puzpuzpuz) [inline internal/serialization/object_data_test.go:217]: nit: what is the reason for this change? Having multiple UTF-8 code point chars is better than a single one.

**Review by puzpuzpuz — APPROVED** [review]

File: internal/serialization/object_data_test.go (hunk 1 above)
Comment (yuce) [reply internal/serialization/object_data_test.go:217]: staticcheck complains about using unicode code instead of some non-printable char. I'll add a few more chars.
